debian/rules

```diff
@@ -59,6 +59,7 @@ override_dh_installinit:
 	install -d -m0755 debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-osd*.conf debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-disk.conf debian/ceph-osd/etc/init
+	install -m0644 build/src/upstart/ceph-osd.conf debian/ceph-osd/etc/init
```

Comment:
@ktdreyer thanks!

@johnlinp but i think we should stop improving upstart script if the change is non-trivial and the benefit is not significant. in this case, we are addressing the users of "make install" and upstart. so i think we can leave as it is. what do you think?